netutil/ip_ranges.go

```diff
@@ -0,0 +1,50 @@
+package netutil
+
+import (
+	"fmt"
+	"net"
+	"strings"
+)
+
+// Default private IP ranges in CIDR notation
+const DefaultPrivateIPRanges = "10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,127.0.0.0/8,169.254.0.0/16,fc00::/7,fe80::/10"
+
+// PrivateCidrRanges holds the default private IP ranges
+var PrivateCidrRanges CidrRanges
```

Comment:
minor, following the go convention for acronyms, 
```suggestion
var PrivateCIDRRanges CIDRRanges
```

Also, I would remove `Ranges`, CIDR is sufficient to describe it.

```suggestion
var PrivateCIDRs CIDRs
```